tests/testthat/_snaps/show_header_names.md

```diff
@@ -86,3 +86,18 @@
       * These values may be dynamically placed into headers (and other locations).
       i Review the `modify_header()` (`?gtsummary::modify()`) help for examples.
 
+# show_header_names() has all values aligned
+
+    Code
+      show_header_names(test_table)
+    Output
+      Column Name   Header                                                                            level*                       N*          n*         p*             
```

Comment:
@ddsjoberg This snapshot doesn't look so great with the wrapping, is there a simpler example that you saw this misalignment in? I can replace it.